Tests/SnapshotConfiguration.swift

```diff
@@ -0,0 +1,64 @@
+// Created by Cal Stephens on 12/15/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+// MARK: - SnapshotConfiguration
+
+/// Snapshot configuration for an individual test case
+struct SnapshotConfiguration {
```

Comment:
Moved this to its own file and added more doc comments -- no real changes otherwise